redfish support for CBS Pending Attribute

ashok2ku opened the pull request:
- Added redfish support CBS Pending Attribute When user apply patch to modify any CBS data CBS attributes will shown in URI- redfish/v1/Systems/system/Bios/SD
- Added support for DIMMs data to persistence

Tested fields: Verified using qemu

Upsteam Status : Inappropriate

supven01 reviewed (commented):
Split the patch into DIMM and BIOS attribute setting.

ashok2ku committed:
redfish support for CBS Pending Attribute

Added redfish support for CBS Pending Attribute,
When user apply patch to modify any CBS data CBS attributes
will shown in URI- redfish/v1/Systems/system/Bios/SD

Tested fields: Verified qemu.

Upsteam Status : Inappropriate

Signed-off-by: ashok2ku <Ashok2.Kumar@amd.com>

ashok2ku committed:
redfish support for DIMMs support

Added support for DIMMs
- POST call to send DIMMs data by BIOS

Tested fields: Verified qemu.

Upsteam Status : Inappropriate

Signed-off-by: ashok2ku <Ashok2.Kumar@amd.com>
